src/main/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionCategoryServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package pl.byczazagroda.trackexpensesappbackend.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.validation.annotation.Validated;
+import pl.byczazagroda.trackexpensesappbackend.dto.FinancialTransactionCategoryCreateDTO;
+import pl.byczazagroda.trackexpensesappbackend.dto.FinancialTransactionCategoryDTO;
+import pl.byczazagroda.trackexpensesappbackend.mapper.FinancialTransactionCategoryModelMapper;
+import pl.byczazagroda.trackexpensesappbackend.model.FinancialTransactionCategory;
+import pl.byczazagroda.trackexpensesappbackend.repository.FinancialTransactionCategoryRepository;
+
+@Service
+@RequiredArgsConstructor
```

Comment:
It's a good idea to add annotations in alphabetical order. This way, in every class or class method, it always looks the same.